File: 1-Iniciante/c/1019_conversao-de-tempo.c
#include <stdio.h>

int main()
{
    int n, horas = 0, minutos = 0, segundos = 0;
    scanf("%d", &n);
    while (n > 0) {
        if (n > 3600) {
            horas++;
            n -= 3600;
        } else if (n > 60) {
            minutos++;
            n -= 60;
        } else {
            segundos = n;
            n = 0;
        }
    }
    printf("%d:%d:%d\n", horas, minutos, segundos);
    return 0;
}

File: 1-Iniciante/python/1019_conversao-de-tempo.py
a = int(input())
horas, minutos, segudos = 0, 0, 0
while a > 0:
    if a > 3600:
        horas += 1
        a -= 3600
    elif a > 60:
        minutos += 1
        a -= 60
    else:
        segundos = a
        a = 0

print(f'{horas}:{minutos}:{segundos}')